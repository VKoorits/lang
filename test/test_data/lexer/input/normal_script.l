nums = [ 1->60 ] // nums = [1,2,3,...,59,60]
for i in nums:
	for j in nums:
		if random.float() > 0.5 :
			swap i, j
/*
	bubble sort, 
	but in random order
*/
print nums
