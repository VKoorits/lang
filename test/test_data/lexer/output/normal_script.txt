ERROR: lex_analyze can`t open file test/test_data/lexer/input/normal_script.l
