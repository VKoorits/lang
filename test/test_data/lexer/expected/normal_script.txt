IDENT_TOKEN: nums
OPERATION_TOKEN: =
OPERATION_TOKEN: [
INT_NUM_TOKEN: 1
OPERATION_TOKEN: ->
INT_NUM_TOKEN: 60
OPERATION_TOKEN: ]
#
IDENT_TOKEN: for
IDENT_TOKEN: i
IDENT_TOKEN: in
IDENT_TOKEN: nums
OPERATION_TOKEN: :
#
IDENT_TOKEN: for
IDENT_TOKEN: j
IDENT_TOKEN: in
IDENT_TOKEN: nums
OPERATION_TOKEN: :
#
IDENT_TOKEN: if
IDENT_TOKEN: random
OPERATION_TOKEN: .
IDENT_TOKEN: float
OPERATION_TOKEN: ()
OPERATION_TOKEN: >
INT_NUM_TOKEN: 0.5
OPERATION_TOKEN: :
#
IDENT_TOKEN: swap
IDENT_TOKEN: i
OPERATION_TOKEN: ,
IDENT_TOKEN: j
#
IDENT_TOKEN: print
IDENT_TOKEN: nums
#
31
